File: Unidad 3/Java/Programa1U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa1u3;

import java.util.Scanner;

public class Programa1U3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Programa para saber si eres mayor o menor de edad");
        int edad;
        
        System.out.println("Introduce tu edad");
        edad=sc.nextInt();
        
        if (edad >= 18){
            System.out.println("Eres mayor de edad");
        
        }else{
            System.out.println("Eres menor de edad");
        }
        
    }
}

File: Unidad 3/C#/Programa1U3.cs
// See https://aka.ms/new-console-template for more information
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa1U3
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Programa para saber si eres mayor o menor de edad");
            int edad;

            Console.WriteLine("Introduce tu edad");
            edad = Convert.ToInt32(Console.ReadLine());

            if (edad >= 18)
            {
                Console.WriteLine("Eres mayor de edad");

            }
            else
            {
                Console.WriteLine("Eres menor de edad");
            }

        }
    }
}